Tests crashing at com.apple.AssetViewer: __64-[ASVInlinePreview preparePreviewOfFileAtURL:completionHandler:]_block_invoke_2

grorg opened the pull request:
#### f91c377541e98376060b37ae823636d7dfff1d75
<pre>
Tests crashing at com.apple.AssetViewer: __64-[ASVInlinePreview preparePreviewOfFileAtURL:completionHandler:]_block_invoke_2
<a href="https://bugs.webkit.org/show_bug.cgi?id=256485">https://bugs.webkit.org/show_bug.cgi?id=256485</a>
rdar://103419649

Unreviewed.

The model element tests are crashing in various ways, causing other tests
to appear to fail. Disable them temporarily.

* LayoutTests/platform/mac/TestExpectations: Skip model-element.

Canonical link: <a href="https://commits.webkit.org/263821@main">https://commits.webkit.org/263821@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/841399efd0488c3ef50f2c93df45d7094b808f46

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/5827 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/5995 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/6184 "Built successfully") | [  ~~🛠 wpe~~](https://ews-build.webkit.org/#/builders/5/builds/7387 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/32/builds/6214 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/6218 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/5962 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/5/builds/7387 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/5933 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/15/builds/5982 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/5281 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 gtk~~](https://ews-build.webkit.org/#/builders/2/builds/7444 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/39/builds/3459 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/10/builds/5256 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/2/builds/7444 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/5326 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/36/builds/5335 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/2/builds/7444 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🛠 tv~~](https://ews-build.webkit.org/#/builders/7/builds/5780 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/17/builds/4746 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [  ~~🛠 tv-sim~~](https://ews-build.webkit.org/#/builders/30/builds/5221 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| | [  ~~🛠 watch~~](https://ews-build.webkit.org/#/builders/4/builds/9342 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/673 "Built successfully and passed tests") | [  ~~🛠 watch-sim~~](https://ews-build.webkit.org/#/builders/31/builds/5582 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
<!--EWS-Status-Bubble-End-->

aproskuryakov commented on LayoutTests/platform/mac/TestExpectations:
```diff
@@ -14,7 +14,6 @@ editing/mac [ Pass ]
 editing/pasteboard/mac [ Pass ]
 fast/dom/Range/mac [ Pass ]
 fast/scrolling/latching [ Pass ]
-model-element [ Pass ]
```
Any more of these to remove?

```
LayoutTests/platform/ios-wk2/TestExpectations:54:model-element/model-element-contents-layer-updates-with-clipping.html [ Pass ]
LayoutTests/platform/ios-wk2/TestExpectations:55:model-element/model-element-contents-layer-updates.html [ Pass ]
LayoutTests/platform/ios/TestExpectations:19:model-element [ Pass ]
LayoutTests/platform/ios/TestExpectations:2211:accessibility/model-element-attributes.html [ Pass ]
LayoutTests/platform/mac/TestExpectations:2106:[ Monterey ] model-element/model-element-graphics-layers-opacity.html [ Pass Failure ]
LayoutTests/platform/mac-wk2/TestExpectations:1472:[ Monterey ] model-element/model-element-source.html [ Pass Crash ]
LayoutTests/platform/mac-wk2/TestExpectations:1722:[ Ventura+ ] model-element/model-element-ready.html [ Pass Failure ]
LayoutTests/platform/mac-wk2/TestExpectations:1723:[ Monterey+ ] model-element/model-element-graphics-layers-opacity.html [ Failure ]
```

grorg committed:
Tests crashing at com.apple.AssetViewer: __64-[ASVInlinePreview preparePreviewOfFileAtURL:completionHandler:]_block_invoke_2
https://bugs.webkit.org/show_bug.cgi?id=256485
rdar://103419649

Unreviewed.

The model element tests are crashing in various ways, causing other tests
to appear to fail. Disable them temporarily.

* LayoutTests/platform/mac/TestExpectations: Skip model-element.

Canonical link: https://commits.webkit.org/263821@main
